### PR TITLE
Istio 0.8 Updates

### PR DESCRIPTION
**Pull request by lincolnthree**

### CODE REVIEW

**Review by geoand — APPROVED** [review]

**Review by gytis — APPROVED** [review]

File: rules/load-balancing-rule.yml [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+  name: spring-boot-istio-routing-service
+spec:
+  hosts:
+  - spring-boot-istio-routing-service
+  http:
+  - route:
+    - destination:
+        host: spring-boot-istio-routing-service
+        subset: a
+      weight: 40
```
Comment (metacosm) [inline rules/load-balancing-rule.yml]: Why did the weight values change?

File: rules/client-gateway-rule.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - client-gateway
+  http:
+  - match:
+    - uri:
+        prefix: /example
+    rewrite:
+      uri: /
+    route:
+    - destination:
+        host: spring-boot-istio-routing-client
```
Comment (metacosm) [inline rules/client-gateway-rule.yml:33]: That host doesn't exist in your descriptors anymore…

**Review by gytis — COMMENTED** [review]
Approved too early...

File: rules/load-balancing-rule.yml (hunk 1 above)
Comment (lincolnthree) [reply rules/load-balancing-rule.yml]: That's an accident. Will fix.

File: rules/client-gateway-rule.yml (hunk 2 above)
Comment (lincolnthree) [reply rules/client-gateway-rule.yml:33]: It does. It's generated by FMP ${project.artifactId} and it's also in the Openshift template, unless I'm mistaken?

Comment (metacosm) [reply rules/client-gateway-rule.yml:33]: Yes, my bad!

**Pull request merged**